### PR TITLE
License

### PR DESCRIPTION
**Pull request by gestchild**
References #1078 / #1138

## Type
✨ Feature  

## Value
Shows licensing information, so users are aware of how they can use the items 

## Screenshot
All possibilities:
screen shot 2017-07-17 at 13 43 22

In use on comic:
screen shot 2017-07-17 at 13 54 48

### All
- [x] Demoed to the relevant people
- [x] PR labelled and assigned
- [x] Any introduced code complexity has been flagged

### UI component only
- [x] A11y tested: `npm run test:accessibility <URL>`
- [x] Browser tested: `npm run test:browsers <URL>`
- [x] Works without JS in the client
- [x] Relevant tracking/monitoring has been considered

### CODE REVIEW

File: server/views/components/license/license.njk [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{% if model.licenseType === 'PDM' %}
+  {% set licenseText = 'Public Domain' %}
+{% else %}
+  {% set licenseText = model.licenseType %}
+{% endif %}
+
+{% set iconClasses = ['icon--2x', 'v-align-middle'] %}
+
+<span class="{{ {s:'LR2', m:'LR2', l:'LR2', xl:'LR2'} | fontClasses }}"{% if model.subject %} about="{{model.subject}}{% endif %}">
```
Comment (davidpmccormick) [inline server/views/components/license/license.njk]: `{s: 'LR2'}` should suffice here now.

File: server/views/components/license/license.njk [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  {% if model.licenseType === 'copyright-not-cleared' %}
+    Copyright Not Cleared
+  {% else %}
+    {% if model.licenseType === 'PDM' %}
+      {% set licenseHref = 'https://creativecommons.org/publicdomain/mark/1.0/' %}
+    {% endif %}
```
Comment (davidpmccormick) [inline server/views/components/license/license.njk]: could these all be `{% elif %}`s?

File: server/views/components/license/license.njk [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    {% if model.licenseType === 'CC-BY' %}
+      {% set licenseHref = 'https://creativecommons.org/licenses/by/4.0/' %}
+    {% endif %}
+    {% if model.licenseType === 'CC-BY-NC' %}
+      {% set licenseHref = 'https://creativecommons.org/licenses/by-nc/4.0/' %}
+    {% endif %}
+    {% if model.licenseType === 'CC-BY-NC-ND' %}
+      {% set licenseHref = 'https://creativecommons.org/licenses/by-nc-nd/4.0/' %}
+    {% endif %}
+    <a rel="license" class="plain-link {{ {s:'LR2', m:'LR2', l:'LR2', xl:'LR2'} | fontClasses }}" href="{{ licenseHref }}">
```
Comment (davidpmccormick) [inline server/views/components/license/license.njk]: same `{s:'LR2'}`

File: server/views/icons/other/cc_zero.svg [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<svg viewBox="-0.5 0.5 64 64">
+<g>
```
Comment (davidpmccormick) [inline server/views/icons/other/cc_zero.svg]: I think you could get rid of all of the `<g>` and </g>` tags in these icons.

File: server/views/components/license/license.njk (hunk 2 above)
Comment (gestchild) [inline server/views/components/license/license.njk]: yep

File: server/views/components/license/license.njk (hunk 1 above)
Comment (gestchild) [inline server/views/components/license/license.njk]: oh yeah, will change

File: server/views/components/license/license.njk (hunk 3 above)
Comment (gestchild) [inline server/views/components/license/license.njk]: 👍

File: server/views/icons/other/cc_zero.svg (hunk 4 above)
Comment (jamesgorrie) [inline server/views/icons/other/cc_zero.svg]: Could we have this in the SVG optimiser?
https://github.com/wellcometrust/wellcomecollection.org/blob/master/server/optimise-svgs.js

**Pull request merged**